plugins/org/src/components/Cards/OwnershipCard/useDirectEntities.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity, UserEntity } from '@backstage/catalog-model';
+import { useApi } from '@backstage/core-plugin-api';
+import {
+  catalogApiRef,
+  formatEntityRefTitle,
+  isOwnerOf,
+} from '@backstage/plugin-catalog-react';
+import qs from 'qs';
+import useAsync from 'react-use/lib/useAsync';
+
+type EntityTypeProps = {
+  kind: string;
+  type: string;
+  count: number;
+};
+
+const getQueryParams = (
+  owner: Entity,
+  selectedEntity: EntityTypeProps,
+): string => {
+  const ownerName = formatEntityRefTitle(owner, { defaultKind: 'group' });
```

Comment:
this makes me curious if it's right, or if it should be `stringifyEntityRef` which always has the whole triplet ... but i assume it's right and matching on the node labels somehow elsewhere

Reply:
this part I'm not change anything because it works as it should be, if using `stringifyEntityRef` I will get `group:default/team-a` instead of `team-a` @freben